src/vue-router/router.js

```diff
@@ -64,6 +75,10 @@ export const router = new Router({
     {
       path: '/',
       name: 'app',
+      // Used to define when App component should render
+      // main app content. If the current route is app route
+      // the content should be rendered.
+      meta: { appRoute: true },
```

Comment:
hmm.. to remove the comment we can rename the appRoute field into something like `isSidebarRendered` or so

Reply:
Updated, renamed to `isNavigationRendered`